dev/example_server.exs

```diff
@@ -49,7 +49,7 @@ defmodule MCPSse.Example.Server do
 end
 
 # Start the MCPSse application
-Application.ensure_all_started(:mcp_sse)
+{:ok, _} = MCP.SSE.start()
```

Comment:
Only question is why we changed this - but I am good as this is an example file!

Reply:
Got it. All good with me!